### DISCUSSION TITLE
unzip with password

### DISCUSSION THREAD

**Discussion by ksvenkatesan77** [cake-build/cake#4005] [Extension requests]
Hi All
Are there any release available to use built-in zip and unzip with password facility?

**Comment by nils-a** [comment]
No, the built-in compression uses `system.io.compression` which IMO does not support encryption.

While not built-in, [Cake.7zip](https://github.com/cake-contrib/Cake.7zip) has support for encrypted zip archives.
[Cake.Compression](https://github.com/akordowski/Cake.Compression/) is IMO based on [SharpZipLib](https://github.com/icsharpcode/SharpZipLib) and could support encryption on zip, but I think it does not.
